smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/providers/extension/MutinyEmitterImpl.java

```diff
@@ -9,6 +9,7 @@
 import io.smallrye.common.annotation.CheckReturnValue;
 import io.smallrye.mutiny.Uni;
 import io.smallrye.mutiny.subscription.Cancellable;
+import io.smallrye.reactive.messaging.EmitterConfiguration;
```

Comment:
Minor: Any reaosn for this import?

Reply:
It receives `EmitterConfiguration` as a constructor parameter.